scripts/babel-plugin-compat-core-import.js

```diff
@@ -0,0 +1,31 @@
+// Convert
```

Comment:
I thought this was just some Node script initially, then I realized it was a tiny Babel plugin and I was like 😮 😮 😮 

Guess I should expect no less of one of Babel's maintainers to just be able to write a Babel plugin so quickly on the fly like that 😅 

Great fix!

Reply:
Sometimes you don't have to look to far to find the right tool for the job!

Reply:
> Sometimes you don't have to look to far to find the right tool for the job!

Hahaha very true!

> I use ASTExplorer to write these tiny plugins

Ooooh that makes a lot of sense. I've seen AST Explorer before, but in context while writing a plugin that is _significantly_ more useful 👀 👀 👀 
Most plugins I'd been thinking of writing would be tiny, for instance, like [literate programming](https://github.com/babel/babel/issues/2592) support (though this would require a [custom parser](https://babeljs.io/docs/en/babel-parser#will-the-babel-parser-support-a-plugin-system) now that I remember where I left off) or [`tsconfig` paths](https://github.com/jaredpalmer/tsdx/issues/91#issuecomment-679541162) support, so that's a really useful tool! Thanks for the example use-case!